docs/extensions-plugins/jupyterlab.mdx

```diff
@@ -0,0 +1,210 @@
+---
+title: Getting Started with Pieces for Developers JupyterLab Extension
+description: Learn how to install the Pieces for JupyterLab Extension via the PYPI Manager, pip, and conda.
+---
+
+<link
+  rel="canonical"
+  href="https://docs.pieces.app/extensions-plugins/jupyterlab"
+/>
+
+export const ReleasePill = props => {
+  return <a target="_blank" style={{textDecoration: "none"}}
+            href="{{ links.installation.jupyterlab }}">
+    <div style={{maxWidth: "fit-content", minWidth: "fit-content"}}>
+      <p style={{
+        padding: "2px 20px",
+        borderRadius: "20px",
+        border: "1px solid #f3f5f6",
+        backgroundColor: "#f3f4f5",
+        color: "black",
+        fontWeight: "500",
+        marginTop: "8px",
+        marginBottom: "15px"
+      }}>{props.children}</p>
+    </div>
+  </a>
+};
+
+export const InstallButton = props => {
+  return <a target="_blank" style={{textDecoration: "none"}} href={ props.src }>
+    <div style={{maxWidth: "fit-content", minWidth: "fit-content"}}>
+      <p style={{
+        padding: "8px 20px",
+        borderRadius: "8px",
+        border: "1px solid #f3f5f6",
+        backgroundColor: "black",
+        color: "white",
+        fontWeight: "500",
+        marginBottom: "8px",
+        marginTop: "25px"
+      }}>{props.children}</p>
+    </div>
+  </a>
+}
+
+<ReleasePill>Compatible with JupyterLab 4.0.0 and higher and is currently in Beta</ReleasePill>
+
+# Your Guide to Getting Started with the Pieces for JupyterLab Extension
+
+_Pieces For Developers captures **NO identifiable user data**. Our **Local only** architecture means your notes never have to leave your device!_
+
+_The <a target="_blank" href="{{ links.website.pfd_desktop_install }}">Pieces Suite</a> must be downloaded and installed separately from the JupyterLab Extensions in order for the Plugin to work properly._
+
+## Steps to Install
+
+<Tabs
+  defaultValue={"pypi"}
+  values={[
+    {label: 'PYPI Manager', value: 'pypi'},
+    {label: 'pip', value: 'pip'},
+    {label: 'conda', value: 'conda'}
+  ]}
+>
+  <TabItem value="pypi">
```

Comment:
we can one line these later it will make it a little smoother and give a full width code block - but we can do it later no need for right now

Reply:
i selected the wrong part of the code here @mason-at-pieces but ill tell you about it tomorrow